woocommerce/woocommerce
# Product Image block: show the setting to control the imageSizing attribute and add the attribute to the `Product Catalog` and `Product Search Results` templates

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





Closes #44689.



### How to test the changes in this Pull Request:



This PR:
- shows the setting to control the `imageSizing` attribute when the user is using a block theme
- adds the `imageSizing` attribute to the product catalog template.

Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

> [!IMPORTANT]  
> Ensure that your shop has products with images of different sizes.

1. Visit the `/wp-admin/site-editor.php?path=%2Fwp_template%2Fall`.
2. Visit `Product Catalog`.
3. Ensure you're using the Products (Beta) block.
4. Ensure that all the images are the same size.
5. Repeat 2-4 steps for `Product Search Results`, `Products by Attribute`, `Products by Category`, `Products by Tag`.
6. Create a new post.
7. Add the Products Beta block.
8. Ensure that all the images are the same size.
9. Create a new post.
10. Add the Product Catalog block
11. Ensure that all the images are the same size.



### Changelog entry




<details>

#### Significance




#### Type




#### Message 
Product Image block: show the setting to control the imageSizing attribute and add the attribute to the Product Catalog and Product Search Results templates.

#### Comment 

</details>
